### PR TITLE
test

### PR DESCRIPTION
**Pull request by bingoarun**
Testing for PR

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import TinyMCE from 'react-tinymce';
 
 class App extends React.Component {
   handleEditorChange = (e) => {
-    console.log('Content was updated:', e.target.getContent());
+    console.log('Editor Content was updated:', e.target.getContent());
```
Comment (openmohan) [inline src/App.js]: [Testing]Editor should be changed to Tinymce editor

**Review by openmohan — COMMENTED** [review]
Review requested

File: src/App.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import TinyMCE from 'react-tinymce';
 
 class App extends React.Component {
   handleEditorChange = (e) => {
-    console.log('Content was updated:', e.target.getContent());
+    console.log('Tinymce Editor Content was updated:', e.target.getContent());
```
Comment (bingoarun) [inline src/App.js:6]: Content changed

**Review by bingoarun — COMMENTED** [review]
Changed to tinymce editor

File: src/App.js (hunk 2 above)
Comment (openmohan) [reply src/App.js:6]: [Testing] LGTM

**Pull request merged**